src/cowboy_swagger.erl

```diff
@@ -70,8 +72,8 @@
 -type metadata() :: trails:metadata(swagger_map()).
 -export_type([swagger_map/0, metadata/0]).
 
--type swagger_version() :: swagger_2_0
-                         | openapi_3_0_0.
+-type swagger_version() :: 'swagger_2_0'
```

Comment:
All these `'` changes are due to `rebar3_lint`'s most recent version.

Reply:
Why don't we change the `regex` in `elvis.config`, instead of adding `'`s?

Reply:
…or adding an `-elvis(…).` attribute here.

Reply:
I personally like the regex as-is, but you might want to be more permissive, sure. Let me know what you prefer.

Reply:
The regex is good _in general_, but it doesn't apply to this particular module. That's fine. We can adjust it for this module. That's what `-elvis(…).` is for, isn't it?

Reply:
😄

Reply:
This occurs in two modules, actually, so I'm going with the "`elvis.config`-based exception" approach.